pom.xml

```diff
@@ -335,4 +367,67 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>locker</id>
```

Comment:
It would be nice to have a comment on what this profile is for and when to use it, if ever 😃